Tests/POEApi.Model.Tests/TestData/SampleInventoryWithQuestItems.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Could you run this through a JSON formatter, such as [this one](https://jsonformatter.curiousconcept.com/#)?